examples/using-gatsby-with-json-yaml/src/pages/index.js

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import { Link } from "gatsby"
+
+const IndexPage = () => (
+  <div>
+    <h1>Sourcing from YAML</h1>
+    <div style={{ maxWidth: `960px`, margin: `1.45rem` }}>
+      <ul>
+        {[
```

Comment:
Why is this list here? Shouldn't the example be sourcing from a YAML file?

Reply:
This list is here because the example code holds both version and if a would be reader, wanted to pick up the code and see it working, if by any chance just opened `http://localhost:8000` it would be presented with something, some content in there, also the list of endpoints that were created. That's my reasoning. Changing this is inconsequential